### ISSUE TITLE
Package tutorials

### ISSUE THREAD

**Issue by abelsiqueira** [JuliaSmoothOptimizers/JSOTutorials.jl#3]
The following packages **need** an "Introduction to" tutorial:

- [ ] LinearOperators.jl (#1)
- [ ] Krylov.jl
- [ ] NLPModels.jl
- [ ] SolverBenchmark.jl
- [ ] JSOSolvers.jl
- [ ] JSOSuite.jl

**Comment by dpo** [comment]
SolverBenchmark already has a pretty good tutorial in the docs. Should it be extracted and moved here? Then we could add a pointer to this repo in the SolverBenchmark docs.

NLPModels also has a good tutorial (that could be expanded).

**Comment by abelsiqueira** [comment]
Yes, I'm thinking most tutorials in the docs should be moved here.
